pytorch_lightning/trainer/trainer.py

```diff
@@ -305,6 +306,10 @@ def __init__(
 
             terminate_on_nan: If set to True, will terminate training (by raising a `ValueError`) at the
                 end of each training batch, if any of the parameters or the loss are NaN or +/-inf.
+                Trainer argument `terminate_on_nan` was deprecated in v1.5 release and will be removed in
+                the v1.7 release. Please use trainer argument `detect_anomaly` instead.
+
```

Comment:
use the docs style or deprecated

```
    .. deprecated:: v1.5 
....
```

check how `reload_dataloaders_every_epoch` is styled above